apps/token-manager/app/src/components/VestingInfoBoxes.js

```diff
@@ -0,0 +1,160 @@
+import React, { useMemo } from 'react'
+import { useConnectedAccount } from '@aragon/api-react'
+import { Box, formatTokenAmount, GU, textStyle, useTheme } from '@aragon/ui'
+import { addressesEqual } from '../web3-utils'
+import { useTotalVestedTokensInfo } from '../app-logic'
+import LocalIdentityBadge from './LocalIdentityBadge/LocalIdentityBadge'
+import TokenIcon from './Icons/TokenIcon'
+import VestingIcon from './Icons/VestingIcon'
+import TransferIcon from './Icons/TransferIcon'
+
+function VestingInfoBoxes({ selectedHolder, tokenSymbol }) {
+  const theme = useTheme()
+  const connectedAccount = useConnectedAccount()
+  const isCurrentUser = addressesEqual(
+    selectedHolder.receiver,
+    connectedAccount
+  )
+  const totalInfo = useTotalVestedTokensInfo(selectedHolder.vestings)
+  return (
+    <React.Fragment>
+      <Box padding={0}>
+        <div
+          css={`
+            padding: ${4 * GU}px;
+            border-bottom: 1px solid ${theme.border};
+          `}
+        >
+          <LocalIdentityBadge
+            entity={selectedHolder.receiver}
+            connectedAccount={isCurrentUser}
+          />
+        </div>
+        <div
+          css={`
+            padding: ${3 * GU}px;
+          `}
+        >
+          <InfoBoxHeader
+            icon={<TokenIcon />}
+            label={'Total Tokens'}
+            content={formatTokenAmount(
+              selectedHolder.holderBalance.balance,
+              18,
```

Comment:
Note that here and below, we shouldn't assume `18` and should be using the token's decimals instead.